apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SoftwareCheckLockedDeviceModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Flex, Popin, Text } from "@ledgerhq/react-ui";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { useTheme } from "styled-components";
+import Animation from "~/renderer/animations";
+import { getDeviceAnimation } from "~/renderer/components/DeviceAction/animations";
+
+export type Props = {
+  isOpen: boolean;
+  deviceModelId: DeviceModelId;
+  productName: string;
+  onClose?: () => void;
+};
+
+const SoftwareCheckLockedDeviceModal = ({ isOpen, deviceModelId, productName, onClose }: Props) => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+
+  return (
+    <Popin position="relative" isOpen={isOpen}>
+      <Popin.Header onClose={onClose}>
+        <></>
+      </Popin.Header>
+      <Flex flexDirection="column" alignItems="center" height="100%" px={8} py={16}>
+        <Text variant="h4Inter" fontWeight="semiBold">
+          {t("syncOnboarding.manual.softwareCheckLockedDeviceModal.title", {
+            deviceName: productName,
+          })}
+        </Text>
+        <Animation
+          animation={getDeviceAnimation(
+            deviceModelId,
+            theme.theme as "light" | "dark",
```

Comment:
I'm surprised you didn't take the chance to create a type and remove all these "light" | "dark" entries

Reply:
true ! i might go in a cleaning spree 🧹

Reply:
So it will be cleaned in https://github.com/LedgerHQ/ledger-live/pull/2242 🤗